content/javascript/concepts/promise/terms/all/all.md

```diff
@@ -56,6 +56,45 @@ Promise.all([promiseA, promiseB])
 The output would be:
 
 ```plaintext
-Results from Promise.all(): [144,42]
+Results from Promise.all(): [23,144]
 Operations for Promise.all() have finished.
 ```
+
+## Codebyte Example
```

Comment:
Looks like the .all() method was erroneously included in multiple issues. This one should be reverted.

Reply:
I reverted my commits on the .all() method @caupolicandiaz